x-pack/plugins/apm/public/index.js

```diff
@@ -27,6 +27,8 @@ import LicenseChecker from './components/app/Main/LicenseChecker';
 
 import { history } from './utils/url';
 
+import { I18nProvider } from '@kbn/i18n/react';
```

Comment:
Same question as on the others ;)
(What is this used for, and do we still need it).

Reply:
`I18nProvider` component is used to setup the i18n context for a tree. After that we are able to use `FormattedMessage` component in order to translate messages. I18nProvider should wrap react app's root component.

We doesn't need to wrap root React component with `I18nProvider` until we use `FormattedMessage`.